scripts/linters/pre_commit_linter.py

```diff
@@ -508,11 +530,22 @@ def main(args=None):
     # third party libraries have their own ways to lint at their fastest
     # (ie. might parallelize on their own)
 
+    err_messages = []
+
     # Concurrency limit: 25.
-    concurrent_task_utils.execute_tasks(tasks_custom, custom_semaphore)
+    custom_task_execution_failed = False
+    try:
+        concurrent_task_utils.execute_tasks(tasks_custom, custom_semaphore)
```

Comment:
I'm not sure whether we should see the exception message during execution or in the `_get_task_output`? Can you please explain?

Reply:
Thanks for pointing it out, I forgot to add before. I have added it now.